Src/Notion.Client/Api/Authentication/RevokeToken/AuthenticationClient.cs

```diff
@@ -1,21 +1,30 @@
+using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Notion.Client
 {
     public sealed partial class AuthenticationClient
-    {   
-        public async Task<HttpStatusCode> RevokeTokenAsync(
+    {
+        public async Task RevokeTokenAsync(
             RevokeTokenRequest revokeTokenRequest,
             CancellationToken cancellationToken = default)
         {
             var body = (IRevokeTokenBodyParameters)revokeTokenRequest;
-            
-            return (await _client.PostAsync(
+
+            var response = await _client.PostAsync<HttpResponseMessage>(
                 ApiEndpoints.AuthenticationUrls.RevokeToken(),
                 body,
                 cancellationToken: cancellationToken
-            )).StatusCode;
+            );
+
+            if (!response.IsSuccessStatusCode)
```

Comment:
I meant this already taken care by the [RestClient](https://github.com/notion-dotnet/notion-sdk-net/blob/acacadfbe2ccc18421da748dabaf26b5a2e8d7ec/Src/Notion.Client/RestClient/RestClient.cs#L159) so don't need to explicitly handle that.